install.php

```diff
@@ -887,32 +887,46 @@ function checkTftpServer() {
     global $extconfigs;
     global $thisInstaller;
     $confDir = $cnf_int->get('ASTETCDIR');
-    // TODO: add option to use external server
-    $remoteFile = "TestFileXXX111.txt";  // should not exist
-    $thisInstaller->tftp_put_test_file();
+    $tftpRootPath = "";
 
+    // TODO: add option to use external server
+    $remoteFileName = ".sccp_manager_installer_probe_sentinel_temp".mt_rand(0, 9999999);
+    $remoteFileContent = "# This is a test file created by Sccp_Manager. It can be deleted without impact";
     $possibleFtpDirs = array('/srv', '/srv/tftp','/var/lib/tftp', '/tftpboot');
+    
+    // write a couple of sentinels to different distro tftp locations in the filesystem
     foreach ($possibleFtpDirs as $dirToTest) {
-        if (file_exists("{$dirToTest}/{$remoteFile}")) {
-            $tftpRootPath = $dirToTest;
-            unlink("{$dirToTest}/{$remoteFile}");
-            outn("<li>" . _("Found ftp root dir at {$dirToTest}") . "</li>");
-            if ($settingsFromDb['tftp_path']['data'] != $tftpRootPath) {
-                $settingsToDb["tftp_path"] =array( 'keyword' => 'tftp_path', 'seq' => 2, 'type' => 0, 'data' => $tftpRootPath);
-                // Need to set the new value here to pass to extconfigs below
-                $settingsFromDb['tftp_path']['data'] = $tftpRootPath;
+        if (is_dir($dirToTest) && is_writable($dirToTest) && empty($tftpRootPath)) {
+            $tempFile = "${dirToTest}/{$remoteFileName}";
+            $FH = fopen($tempFile, "w");
```

Comment:
file_put_contents($tempFile, $remoteFileContent);

is simpler?

Reply:
I am a C guy, and it shows :-) Will use your recommendation.